examples/imitation_learning_openbot/generate_dataset.py

```diff
@@ -197,7 +160,7 @@
             action, policy_step_info = policy.step(obs)
```

Comment:
- rename `hit_obstacle = False` to `collect_data_for_episode = True` to improve readability

Reply:
Updated this